app/internal/page/fixtures.go

```diff
@@ -67,3 +105,201 @@ func MakeCertificateProvider(firstNames string) actor.CertificateProvider {
 		RelationshipLength:      "gte-2-years",
 	}
 }
+
+func CompleteDonorDetails(lpa *Lpa) {
+	lpa.You = MakePerson()
+	lpa.WhoFor = "me"
+	lpa.Type = "pfa"
+	lpa.Tasks.YourDetails = TaskCompleted
+}
+
+func AddAttorneys(lpa *Lpa, count int) []string {
+	if count > len(AttorneyNames) {
+		count = len(AttorneyNames)
+	}
+
+	var firstNames []string
+	for i := 0; i < count; i++ {
```

Comment:
if you change those maps to slices these could be neat `for name := range AttorneyNames[:count] { ... }`